### PR TITLE
Update docbook-xsl-nons to 2020-06-03.

### PR DESCRIPTION
**Pull request by satmandu**
##
Builds:
- [x] `x86_64`
- [x] `i686`
- [x] `armv7l`
##

##
### Run the following to get this pull request's changes locally for testing.
```bash
CREW_REPO=https://github.com/satmandu/chromebrew.git CREW_BRANCH=docbook crew update \
&& yes | crew upgrade
```

### CODE REVIEW

File: packages/docbook_xsl_nons.rb [hunk 1]
```diff
@@ -3,37 +3,46 @@
...
   description 'The DocBook XSL Stylesheets package contains XSL stylesheets. These are useful for performing transformations on XML DocBook files.'
   homepage 'https://github.com/docbook/xslt10-stylesheets'
-  @_ver = '1.79.2'
-  version "#{@_ver}-3"
+  version '1.79.2-2020-06-03'
   license 'custom'
   compatibility 'all'
-  source_url "https://github.com/docbook/xslt10-stylesheets/releases/download/release/#{@_ver}/docbook-xsl-nons-#{@_ver}.zip"
-  source_sha256 'ba41126fbf4021e38952f3074dc87cdf1e50f3981280c7a619f88acf31456822'
+  source_url 'https://github.com/docbook/xslt10-stylesheets/releases/download/snapshot/2020-06-03/docbook-xsl-nons-snapshot.zip'
```
Comment (uberhacker) [inline packages/docbook_xsl_nons.rb:9]: Change `2020-06-03` to `#{version.split('-')[1]}`?

Comment (uberhacker) [reply packages/docbook_xsl_nons.rb:9]: Actually, that won't work.

Comment (uberhacker) [reply packages/docbook_xsl_nons.rb:9]: I wouldn't suggest this but it works.
```
version.split('-')[1..3].join('-')
```

**Comment by satmandu** [comment]
This changes nothing, and there's no reason to build from scratch, so I'm closing this.

**Pull request closed without merging**